devops/scripts/benchmarks/benches/compute.py

```diff
@@ -264,6 +264,24 @@ def parse_unit_type(compute_unit):
 
 
 class ComputeBenchmark(Benchmark):
+
+    # List of benchmarks that should not be traced by Unitrace
+    not_traceable = [
+        # timeouts with v2 adapter
```

Comment:
But why does it timeout? Like I said previously, we should reduce the number of iterations when running under unitrace.

Reply:
I meant internal benchmark iterations, for example:
https://github.com/intel/llvm/blob/sycl/devops/scripts/benchmarks/benches/compute.py#L468

If you were to trace that, you'd get a massive file with mostly identical sections of kernel/finish invocations. During tracing it rarely makes sense to do more than, I don't know, 10 iterations (to account for warmup and whatnot).